pkg/controller/gitopsconfig/jobmonitor.go

```diff
@@ -188,7 +193,7 @@ func (job *jobmonitor) checkResult() (jobState gitopsv1alpha1.GitOpsConfigStatus
 		log.Info("retrying the GetJob for CheckResult")
 		jobOutput, err = job.getjobmonitor()
 		if err != nil {
-			log.Error(err, "Error in GetJob for CheckResult")
```

Comment:
Why change Error to Info?

Reply:
I don't find anything error-like in this case. I.e. for me this is a rather normal case which can happen and user should be notified of it with an information, not error